cmd/checkmetrics/csv.go

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/csv"
+	"errors"
+	"math"
+	"os"
+	"strconv"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/montanaflynn/stats"
+)
+
+// csvRecord holds the data for a single 'test'. Some of the data is imported
```

Comment:
Yeah, that would be nicer. As it happens, it seems `Name` is un-used, so I've dropped that, and as that only leaves `Records` as something imported from the file, in this case I've commented and laid it out to make that clearer. A new struct for just `Records` now felt a little overkill